src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,15 +1,24 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinTable;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "comments")
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @OneToMany(fetch = FetchType.EAGER)
```

Comment:
OneToMany or ManyToMany? Also, it's a bad practice to set Eager fetch type for toMany relations.

Reply:
Users have many comments
Comments have many users?